nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -428,12 +429,13 @@ IEnumerator CoInitializeSecondWidget()
                 _commandLineOptions.AppProtocolVersion,
                 out var appProtocolVersion);
             Widget.Find<VersionSystem>().SetVersion(appProtocolVersion);
-            Widget.Find<GrayLoadingScreen>().ShowProgress(GameInitProgress.ProgressCompleted);
-            ShowNext(agentInitializeSucceed);
             Analyzer.Instance.Track("Unity/Intro/Start/ShowNext");
 
             StartCoroutine(CoUpdate());
             ReservePushNotifications();
+
+            yield return new WaitForSeconds(2f);
```

Comment:
What is this waiting for?

Reply:
it's waiting for last tweening in GrayLoadingScreen, when progress is 100

Reply:
I will add comment that description to code